swarm/pss/pss_test.go

```diff
@@ -475,6 +482,196 @@ func testAsymSend(t *testing.T) {
 	}
 }
 
+// params in run name:
+// nodes/msgs/addrbytes/adaptertype
+// if adaptertype is exec uses execadapter, simadapter otherwise
+func TestNetwork(t *testing.T) {
+	t.Run("8/1024/2/sim", testNetwork)
+}
+
+func testNetwork(t *testing.T) {
+
+	type msgnotifyC struct {
+		id     discover.NodeID
+		msgIdx int
+	}
+	topic := whisper.BytesToTopic([]byte("foo:42"))
+	hextopic := common.ToHex(topic[:])
+
+	paramstring := strings.Split(t.Name(), "/")
+	nodecount, _ := strconv.ParseInt(paramstring[1], 10, 0)
+	msgcount, _ := strconv.ParseInt(paramstring[2], 10, 0)
+	addrsize, _ := strconv.ParseInt(paramstring[3], 10, 0)
+	messagedelayvarianceusec := (int(msgcount)/1000 + 1) * 1000 * 1000
```

Comment:
It is idiomatic Go to use `mixedCaps` (see [here](https://golang.org/doc/effective_go.html#mixed-caps)), so `messageDelayVarianceUseC`.